lib/split/algorithms/whiplash.rb

```diff
@@ -14,6 +13,7 @@ def choose_alternative(experiment)
         private
 
         def arm_guess(participants, completions)
+          raise "This method will not work as SimpleRandom was removed due to licensing."
```

Comment:
I have to question whether this is an 'exceptional' event or something we're prepared to handle by simply returning a failed response.

Reply:
Agreed. To be honest these `raise`s were the first thing I thought of when thinking _we want this to be obvious to the user_.

I think that alerting the user somehow when they have set an experiment to use `whiplash` algorithm would be the best time and place to guard against this code path.